README.md

```diff
@@ -1 +1,23 @@
-# earthly-intellij-plugin
\ No newline at end of file
+# earthly-intellij-plugin
+
+IntelliJ plugin for Earthly language support.
+
+## Features
+- [x] Syntax highlighting for Earthfiles
+- [ ] Code completion
+- [ ] Formatter
+- [ ] Commenter
+- [ ] Go To Symbol Contributor
+- [ ] ... Let us know!
+
+## Testing
+### Running an IDE instance
+1. Run `./gradlew runIde`. This will open an IDE instance with the plugin installed.
+2. Create a new Java project
+3. Create a new file named `Earthfile` and paste the contents of your choice there.
+
+### Running jUnit tests
+1. Run `./gradlew test --tests LexerTests`
```

Comment:
Can you make a basic Earthfile that can run this test? :-)

It might make sense to also hook up GitHub Actions to run it continuously.

(This could also be done in a separate PR - it doesn't have to be this one)